draft-ietf-quic-reliable-stream-reset.md

```diff
@@ -148,6 +175,23 @@ SHOULD NOT be retransmitted.
 As described in (Section 3.2 of {{RFC9000}}), it MAY deliver data beyond that
 offset to the application.
 
+# Aborting reading
+
+Using the STOP_SENDING frame, {{!RFC9000}} allows the receiver of a stream to
+abort reading and to request the sender to reset the stream. Analogously, the
+ENOUGH frame can be used to request resetting the stream at a specific offset.
+Sending ENOUGH is permitted in all stream states that allow sending STOP_SENDING
+(see section 3.2 of {{!RFC9000}}).
+
+The receiver of an ENOUGH frame MUST reset the stream by sending either a
+RESET_STREAM frame or a CLOSE_STREAM frame, if the stream is in the "Ready" or
+"Send" state. It MAY honor the receiver's request to reliably transmit the
+specified number of bytes, but it MAY choose any other offset as well.
+
+Note that if the sender has already sent a CLOSE_STREAM frame, it might not be
+possible to send another CLOSE_STREAM with a larger offset (see
+{{multiple-frames}}) for details.
+
 ## Multiple CLOSE_STREAM / RESET_STREAM frames {#multiple-frames}
```

Comment:
This section was a subsection of "Resetting Streams" but is now part of "Aborting reading."

I think that's an error and that we might want to move this subsection up, into "Resetting Streams"?